install/templates/prometheus.tmpl

```diff
@@ -28,16 +28,7 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 networks:
-  # Bridge so node-exporter can get the real NIC details
-  # See https://stackoverflow.com/a/66689508 for more info
   monitor-net:
-    driver: bridge
-    ipam:
-      driver: default
-      config:
-        - subnet: 172.23.0.0/16
```

Comment:
The [RocketPool docs](https://docs.rocketpool.net/guides/node/grafana#enabling-the-metrics-server) don't assume any particular network address for monitoring, so docker compose can assign network address for monitor-net by itself. The network address should not change later during RP node operation, since the networks are not recreated on starting/stopping RocketPool services.